sdk/purview/Azure.Analytics.Purview.Share/tests/Samples/Sample01_CreateSentShare.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+#region Snippet:Azure_Analytics_Purview_Share_Samples_01_Namespaces
+using Azure.Core;
+using Azure.Identity;
+#endregion Snippet:Azure_Analytics_Purview_Share_Samples_01_Namespaces
+
+namespace Azure.Analytics.Purview.Share.Samples
+{
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1649:File name should match first type name", Justification = "For documentation purposes")]
+    internal class CreateSentShare
+    {
+        public async Task CreateSentShareSample()
+        {
+            #region Snippet:Azure_Analytics_Purview_Share_Samples_CreateSentShare
+            var credential = new DefaultAzureCredential();
+            var endPoint = "https://<my-account-name>.purview.azure.com";
```

Comment:
```suggestion
#if SNIPPET
            var credential = new DefaultAzureCredential();
            var endpoint = "https://<my-account-name>.purview.azure.com";
#else
            var credential = TestEnvironment.Credential;
            var endPoint = TestEnvironment.Endpoint;
#endif
```

Once these become tests, this pattern will help control configuration that should appear in snippets when injected versus those used when executing tests.   The intent is to keep these conditionals as small and isolated as possible to ensure that the majority of code runs as part of the test.